src/pymatgen/analysis/local_env.py

```diff
@@ -52,16 +52,15 @@
 __date__ = "August 17, 2017"
 
 MODULE_DIR = os.path.dirname(os.path.abspath(__file__))
-yaml = YAML()
 
-with open(f"{MODULE_DIR}/op_params.yaml") as file:
-    default_op_params = yaml.load(file)
+with open(f"{MODULE_DIR}/op_params.yaml", encoding="utf-8") as file:
+    DEFAULT_OP_PARAMS: dict[str, dict[str | int, float] | None] = YAML().load(file)
 
-with open(f"{MODULE_DIR}/cn_opt_params.yaml") as file:
-    cn_opt_params = yaml.load(file)
+with open(f"{MODULE_DIR}/cn_opt_params.yaml", encoding="utf-8") as file:
+    CN_OPT_PARAMS: dict[int, dict[str, list[str | dict[str, float]]]] = YAML().load(file)
```

Comment:
@shyuep Apologize, I didn't realize [this recording would be needed externally](https://github.com/materialsproject/atomate2/pull/1030), I would add it back with a deprecation period.